### DISCUSSION TITLE
GPTH runs through only goes through the dupes and checking the photos but it exits before copying the photos.

### DISCUSSION THREAD

**Discussion by eedbarr** [TheLastGimbus/GooglePhotosTakeoutHelper#219] [I NEED HELP] [answered]
Hello. Per the title of the post, GPTH would run through the prompts but when it goes through the duplicate finding and checking of the photos, I noticed it just exits out. 

I have 300GB of photos and I have extracted them all in one Takeout folder. Is there something I'm missing to have it run properly? Thank you for this script and I hope I get to transfer my photos to Synology.

**Comment by TheLastGimbus** [comment]
Show me the output of what is happening, otherwise i really cant help

**Comment by eedbarr** [comment]
sorry it took me a while. I tried redoing gpth with the new update and this is what I get:
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/22019514/c64255d7-ed3f-40a5-a482-6b68de1e7e64)
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/22019514/bda577f7-d4e7-4241-b402-85736a645e3b)


Not sure if I should try something else.

**Reply by TheLastGimbus** [reply, marked as answer]
Ah damn, we have a #251 here

You wll need to do as i said in https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/issues/251#issuecomment-1747006340

sorry for inconvinience :/
